store/store_action.go

```diff
@@ -347,6 +358,10 @@ func (s *Store) snapAction(ctx context.Context, currentSnaps []*CurrentSnap, act
 			CohortKey:        curSnap.CohortKey,
 			ValidationSets:   valsetKeys,
 		}
+		// `held` field was introduced in version 54 https://api.snapcraft.io/docs/
+		if len(curSnap.HeldBy) > 0 && minStoreVersion(storeVer, 54) {
```

Comment:
Then this would be explicit about defaulting to using the held-by when no version if specified

```suggestion
		if len(curSnap.HeldBy) > 0 && (storeVer <= 0 || storeVer >= 54) {
```

Reply:
Yes, I agree. Even if we end up using a pointer this would be more explicit `(storeVer == nil || *storeVer >= 54)`. Thank you!